internal/common/shared/iigoroles.go

```diff
@@ -41,8 +41,9 @@ func (r Role) MarshalJSON() ([]byte, error) {
 // RuleVoteType provides enumerated values for Approving, Rejecting or Abstaining from a vote.
 type RuleVoteType int
 
+// type of vote for each ballot
 const (
-	Approve = iota
+	Approve RuleVoteType = iota
```

Comment:
ah, thanks for this. can you submit this in a separate PR?

Reply:
Check PR #372

Reply:
PR #372